packages/itwinui-react/src/core/Checkbox/Checkbox.tsx

```diff
@@ -36,6 +36,14 @@ type CheckboxProps = {
    * @default false
    */
   isLoading?: boolean;
+  /**
+   * Passes properties for checkbox wrapper.
+   */
+  wrapperProps?: React.ComponentProps<'label'>;
+  /**
+   * Passes properties for checkbox label.
+   */
+  labelProps?: React.ComponentProps<'label'>;
```

Comment:
Since these props are added to a `span` element, make sure this prop's type changes from `<'label'>` to `<'span'>`

Reply:
Done!